README.md

```diff
@@ -81,6 +81,27 @@ Zarf makes DevSecOps for air gap possible.
   </td>
   </tr>
 <!-- row end -->
+
+<!-- row start -->
+  <tr valign="top">
+  <td>
+
+  **Join Slack Channels**
+
+  </td>
+  <td>
+
+  Join our project channels on K8's slack. #zarf #zarf-dev
```

Comment:
```suggestion
  Join our project channels on K8's slack. [#zarf](https://kubernetes.slack.com/archives/C03B6BJAUJ3) [#zarf-dev](https://kubernetes.slack.com/archives/C03BP9Z3CMA)
```

Reply:
The markdown-style links might not work since we're doing HTML here. Might need `<a>` tags